File: src/main/java/no/nav/domain/pensjon/regler/beregning2011/BeregningsResultatAlderspensjon2011.java
package no.nav.domain.pensjon.regler.beregning2011;

/**
 * @author Magnus Bakken (Accenture), PK-20716
 */
public class BeregningsResultatAlderspensjon2011 extends AbstraktBeregningsResultat {


    /**
     * Informasjon om pensjon under utbetaling, regnet uten gjenlevenderettighet.
     * Kommer ikke til utbetaling, da denne kun er regnet ut som del av beregningen av gjenlevendetillegget på AP2016.
     */
    private PensjonUnderUtbetaling pensjonUnderUtbetalingUtenGJR;

    private BeregningsInformasjon beregningsInformasjonKapittel19;

    private BeregningsInformasjon beregningsInformasjonAvdod;

    private AldersberegningKapittel19 beregningKapittel19;

    private boolean PREG_nullstilt;


}

File: src/main/kotlin/no/nav/pensjon/regler/domain/beregning2011/BeregningsResultatAlderspensjon2011.kt
package no.nav.pensjon.regler.domain.beregning2011

class BeregningsResultatAlderspensjon2011 : AbstraktBeregningsResultat() {
    /**
     * Informasjon om pensjon under utbetaling, regnet uten gjenlevenderettighet.
     * Kommer ikke til utbetaling, da denne kun er regnet ut som del av beregningen av gjenlevendetillegget på AP2016.
     */
    var pensjonUnderUtbetalingUtenGJR: PensjonUnderUtbetaling? = null
    var beregningsInformasjonKapittel19: BeregningsInformasjon? = null
    var beregningsInformasjonAvdod: BeregningsInformasjon? = null
    var beregningKapittel19: AldersberegningKapittel19? = null

}